### PR TITLE
[Refactor:RainbowGrades] Refactor getDisplay

### PR DESCRIPTION
**Pull request by ziesski**
Inline the $isUsed variable inside the array directly, instead of declaring it before. 
More concise, enhance readability.

### CODE REVIEW

**Review by justin-sadakhom — CHANGES_REQUESTED** [review]
Good uses of variable names, ternary operator, and inline function call to make the code more readable. See my comment for a suggestion.

**Review by ArnavPawar — APPROVED** [review]
looks fine. Just doing the same code but changing the variable names. Could be less lines but its fine.

**Review by ArnavPawar — CHANGES_REQUESTED** [review]
correction ^ just fix changes from justin and it should be good

File: site/app/models/RainbowCustomization.php [hunk 1]
```diff
@@ -404,24 +404,19 @@ public function getBenchmarkPercent() {
      * @return array<int, array<string, bool|string>> multidimensional array of display option data
      */
     public function getDisplay(): array {
-        // Get allowed benchmarks
-        $display = RainbowCustomizationJSON::allowed_display;
-        $retArray = [];
+        $allowedDisplayOptions = RainbowCustomizationJSON::allowed_display;
```
[the remark this answers is not in the document]
Comment (ziesski) [reply site/app/models/RainbowCustomization.php:407]: good catch, thanks!

File: site/app/models/RainbowCustomization.php [hunk 2]
```diff
@@ -404,24 +404,19 @@ public function getBenchmarkPercent() {
...
-            $usedDisplay = [];
+        $used_display_options = $this->RCJSON ? $this->RCJSON->getDisplay() : [];
 
-        // Add data into retArray
-        foreach ($display as $display_option) {
-            in_array($display_option, $usedDisplay) ? $isUsed = true : $isUsed = false;
-
-            // Add display to return array
-            $retArray[] = ['id' => $display_option, 'isUsed' => $isUsed];
+        foreach ($allowed_display_options as $optionId) {
```
Comment (justin-sadakhom) [inline site/app/models/RainbowCustomization.php:412]: You missed `$optionId`

**Review by justin-sadakhom — CHANGES_REQUESTED** [review]
Sorry, just one last variable left to change

**Review by williamjallen — APPROVED** [review]

**Pull request merged**